### ISSUE TITLE
Table "django_content_type" missing when running migrate

### ISSUE THREAD

**Issue by Chessmasterrr** [fsinfuhh/Bitpoll#55]
First of all, thanks for your great work!

When I try to install your application in release version 0.9 and run the `./manage.py migrate` command, I get the following error:
`django.db.utils.OperationalError: no such table: django_content_type`

**How to reproduce:**
- I am trying to run bitpoll in a [docker container](https://hub.docker.com/repository/docker/chessmasterrr/bitpoll) with this [Dockerfile](https://github.com/Chessmasterrr/docker/blob/master/bitpoll/Dockerfile).
- Run this container: `docker run -p 8000:8000 -v /path/on/host/settings_local.py:/Bitpoll/bitpoll/settings_local.py -v /path/on/host/database/:/Bitpoll/database/ -d chessmasterrr/bitpoll:latest`
- Try to generate a database via `docker exec <containerID> ./manage.py migrate` resulting in this error trace:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 383, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: django_content_type

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/migrate.py", line 87, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/executor.py", line 18, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/loader.py", line 49, in __init__
    self.build_graph()
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/loader.py", line 206, in build_graph
    self.load_disk()
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/loader.py", line 108, in load_disk
    migration_module = import_module(migration_path)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/django_token_bucket/migrations/0002_auto_20191216_1228.py", line 10, in <module>
    class Migration(migrations.Migration):
  File "/usr/local/lib/python3.8/site-packages/django_token_bucket/migrations/0002_auto_20191216_1228.py", line 26, in Migration
    field=models.ForeignKey(default=ContentType.objects.get_for_model(get_user_model()).id, on_delete=django.db.models.deletion.CASCADE, to='contenttypes.ContentType'),
  File "/usr/local/lib/python3.8/site-packages/django/contrib/contenttypes/models.py", line 51, in get_for_model
    ct = self.get(app_label=opts.app_label, model=opts.model_name)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 402, in get
    num = len(clone)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 256, in __len__
    self._fetch_all()
  File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 1242, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 55, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/compiler.py", line 1133, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.8/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 383, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: django_content_type
```

Can you give me a hint what I am doing wrong?
If any further information are needed I will be happy to provide them.

**Comment by Akasch** [comment]
Is this a new database or a migration of an existing database?

I have an Idea that is happening: The dependencies of a migration in the django_token_bucket seems to be missing. I will try to fix it tomorrow.

**Comment by Chessmasterrr** [comment]
Thanks for your comment!
It is a complete new database.

**Comment by Akasch** [comment]
I have a fix for the root cause. Once https://github.com/fsinfuhh/django_token_bucket/pull/3 is merged, i will bump the version here and release 0.9.1 (and probably 0.10 with the new features from master)

PS: I have seen you are using sqlite in your Dokerfile, with sqlite the invitation of Users into Groups does not work, so maybe you should set GROUP_MANAGEMENT = False in the settings

**Comment by Akasch** [comment]
This should be fixed with version 0.9.1

**Comment by Chessmasterrr** [comment]
Thanks for your fast help! It is now working.

PS: I will adapt my settings, thanks for the hint!
